tests/test_trailedEdgeSources.py

```diff
@@ -0,0 +1,220 @@
+#
+# This file is part of meas_extensions_trailedSources.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import numpy as np
+import unittest
+import lsst.utils.tests
+import lsst.meas.extensions.trailedSources
+from lsst.meas.base.tests import AlgorithmTestCase
+from lsst.utils.tests import classParameters
+
+# Trailed-source length, angle, and centroid, one on the edge, one not.
+Ls = np.array([5, 5])
```

Comment:
Suggest more descriptive variable names so you can just say "one trailed source is on the edge and one is not." e.g., `trail_lengths`, `trail_angles`, `trail_x_centroids`, `trail_y_centroids`. If you really want to keep most of these and only change one, `Ls` is by far my least favorite 😄